deploy/docker/entrypoint.sh

```diff
@@ -1,5 +1,19 @@
 #!/usr/bin/env bash
 
+if [ $FILESTORE_IP_ADDRESS ];
+then
```

Comment:
The `if` check is incorrect. I think we need `[[ -n "$FILESTORE_IP_ADDRESS" ]]` to check if it's non-empty.

```suggestion
if [[ -n "$FILESTORE_IP_ADDRESS" ]]; then
```